collect_app/src/main/java/org/odk/collect/android/formlists/blankformlist/DeleteBlankFormFragment.kt

```diff
@@ -23,6 +23,7 @@ class DeleteBlankFormFragment(
 
     private lateinit var blankFormListViewModel: BlankFormListViewModel
     private lateinit var multiSelectViewModel: MultiSelectViewModel
+    private var allSelected = false
```

Comment:
We could drop using a property like this if we had a `zip2` transformation on `LiveData` (that would let us combine the selected and list states). I did some work on improving our zip helpers in #5521, so don't want to duplicate that here.